New log file opened at Fri Oct 02 17:57:58 CEST 2015 by 7136@Toshiba-Robin
Taking next connection to: 53879
New client attempting to connect on port: 53880-53879
  Address = /145.97.175.9:53879
This client on port 53880 has uuid be2b16ca-6d12-4673-845a-38adbdcfce4f configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-be2b16ca-6d12-4673-845a-38adbdcfce4f
Taking next connection to: 53879
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53879
New client attempting to connect on port: 53971-53879
  Address = /145.97.175.9:53879
Handshake exception on socket: 53971
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53879
New client attempting to connect on port: 53972-53879
  Address = /145.97.175.9:53879
This client on port 53972 has uuid 5f3248e5-bf88-4989-88dd-ea8262760469 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-be2b16ca-6d12-4673-845a-38adbdcfce4f, play-fork-run-5f3248e5-bf88-4989-88dd-ea8262760469
Taking next connection to: 53879
